src/category_theory/limits/cone_category.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2021 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+
+import category_theory.limits.preserves.shapes.terminal
+
+/-!
+
+# Limits and the category of (co)cones
+
+This files contains results that stem from the limit API. For the definition and the category
+instance of `cone`, please refer to `category_theory/limits/cones.lean`.
+
+A cone is limiting iff it is terminal in the category of cones. As a corollary, an equivalence of
+categories of cones preserves limiting properties. We also provide the dual.
+
+-/
+
+namespace category_theory.limits
+
+open category_theory
+
+universes v u
+
+variables {J : Type v} [category.{v} J] {J' : Type v} [category.{v} J']
+variables {C : Type u} [category.{v} C] {C' : Type u} [category.{v} C']
+
+/-- A cone is a limit cone iff it is terminal. -/
+def cone_limiting_iff_terminal {F : J ⥤ C} (c : cone F) : is_limit c ≃ is_terminal c :=
+is_limit.iso_unique_cone_morphism.to_equiv.trans
+{ to_fun := λ h, by exactI is_terminal.of_unique _,
+  inv_fun := λ h s, ⟨⟨is_terminal.from h s⟩, λ a, is_terminal.hom_ext h a _⟩,
+  left_inv := by tidy,
+  right_inv := by tidy }
+
+lemma limit_cone_lift_eq_terminal_from {F : J ⥤ C} (c : cone F) (hc : is_limit c) (s : cone F) :
```

Comment:
Do you want `c` to be explicit? It is determined by `hc`. I think this could be called
```suggestion
lemma is_limit.lift_cone_morphism_eq_is_terminal_from {F : J ⥤ C} (c : cone F) (hc : is_limit c) (s : cone F) :
```